addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
I believe we agreed in Slack before that single line was fine with a reasonable wrap - purely the judgement call of the person writing the file imo.

There are cases with nested destructuring where a single line doesn't look as good in my opinion, e.g.

```
const {
  Component,
  computed: {
    readOnly
  },
  run
} = Ember
```
vs
`const { Component, computed : { readOnly },  run } = Ember`

But I do believe this is a matter of preference and that we can have a few heuristics to guide these styles.